validator/src/main/resources/expected-data-template/spark/sparkAppExpectedAWSSDKTrace.mustache

```diff
@@ -16,35 +16,7 @@
     {
       "fault": false,
       "error": false,
-      "throttle": false,
-      "http": {
-        "request": {
```

Comment:
Why are we removing all of it?

Reply:
the final one is 

```
[{
  "name": "aws-otel-integ-test",
  "fault": false,
  "error": false,
  "throttle": false,
  "http": {
    "request": {
      "url": "{{endpoint}}/aws-sdk-call",
      "method": "GET"
    },
    "response": {
      "status": 200
    }
  },
  "subsegments": [
    {
      "fault": false,
      "error": false,
      "throttle": false
    }
  ]
}]
```

the http block under subsegments are unpredictable, some times a put request is appeared